ddocs/medic-client/views/contacts_by_reference/map.js

```diff
@@ -5,7 +5,8 @@ function(doc) {
     doc = doc.tombstone;
   }
 
-  if (doc.type === 'clinic' ||
+  if (doc.type === 'contact' ||
+      doc.type === 'clinic' ||
       doc.type === 'health_center' ||
       doc.type === 'district_hospital' ||
       doc.type === 'national_office' ||
```

Comment:
Hmm yeah. I think it was a 4th place type someone starting adding, but it was never used throughout the app. I've left it here just in case someone is using it. It'll be removed once and for all when we delete the backwards compatibility code (hopefully in 4.0).